build/rpmfc.c

```diff
@@ -402,7 +402,19 @@ static int getOutputFrom(ARGV_t argv,
 
 reap:
     /* Collect status from prog */
-    reaped = waitpid(child, &status, 0);
+
+    do {
+	reaped = waitpid(child, &status, 0);
+    } while (reaped == -1 && errno == EINTR);
+
+    // It's not allowed to call WIFEXITED or WEXITSTATUS if waitpid return -1.
+    // Note, all bits set, since -1 == 0xFFFFFFFF
```

Comment:
See https://github.com/rpm-software-management/rpm/blob/master/CONTRIBUTING.md#comments for comment style.

Reply:
Fixed